packages/node_modules/@ciscospark/widget-space/src/index.html

```diff
@@ -4,56 +4,51 @@
 <head>
   <title>Space Widget Demo</title>
   <meta charset="utf8"/>
-  <script nomodule src="https://cdnjs.cloudflare.com/ajax/libs/babel-polyfill/6.26.0/polyfill.min.js" integrity="sha256-WRc/eG3R84AverJv0zmqxAmdwQxstUpqkiE+avJ3WSo=" crossorigin="anonymous"></script>
 </head>
 <body>
 <div style="float: left;">
-  <h2>Browser Globals</h2>
+  <h2>Space Widget</h2>
```

Comment:
This is what gets served when you run `npm run serve:package widget-space`. It is a good way to check styles outside of the widget-demo. We've had the demo add styles that the individual widgets did not get and a bug was missed in the past. I've made this page a little more user friendly and less relying on our original env vars.